scene/2d/path_2d.cpp

```diff
@@ -201,7 +479,21 @@ void Path2D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("set_curve", "curve"), &Path2D::set_curve);
 	ClassDB::bind_method(D_METHOD("get_curve"), &Path2D::get_curve);
 
+	ClassDB::bind_method(D_METHOD("set_debug_enabled", "enabled"), &Path2D::set_debug_enabled);
+	ClassDB::bind_method(D_METHOD("get_debug_enabled"), &Path2D::get_debug_enabled);
+
+	ClassDB::bind_method(D_METHOD("set_debug_custom_enabled", "enabled"), &Path2D::set_debug_custom_enabled);
+	ClassDB::bind_method(D_METHOD("get_debug_custom_enabled"), &Path2D::get_debug_custom_enabled);
+
+	ClassDB::bind_method(D_METHOD("set_debug_custom_color", "debug_custom_color"), &Path2D::set_debug_custom_color);
+	ClassDB::bind_method(D_METHOD("get_debug_custom_color"), &Path2D::get_debug_custom_color);
+
 	ADD_PROPERTY(PropertyInfo(Variant::OBJECT, "curve", PROPERTY_HINT_RESOURCE_TYPE, "Curve2D", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_EDITOR_INSTANTIATE_OBJECT), "set_curve", "get_curve");
+
+	ADD_GROUP("Debug", "debug_");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "debug_enabled"), "set_debug_enabled", "get_debug_enabled");
+	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "debug_custom_enabled"), "set_debug_custom_enabled", "get_debug_custom_enabled");
```

Comment:
This property only controls the color. Why is it needed? You can use transparent color as default.

Reply:
Also https://github.com/godotengine/godot/pull/105606/files#r2052875614